bespin/option_spec/stack_objs.py

```diff
@@ -22,6 +22,12 @@
 import os
 import re
 
+# python 2.6 set compat
+try:
+    set
+except NameError:
+    from sets import Set as set
```

Comment:
I'm pretty sure 2.6 has set.

2.6.9 certainly does (that's the 2.6 installed on my laptop)